tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py

```diff
@@ -55,11 +52,55 @@ def validate_environment(is_local: bool, use_remote_secrets: bool):
         )
 
 
+def render_report_output_prefix(ctx: click.Context) -> str:
+    """Render the report output prefix for any command in the Connector CLI.
+
+    The goal is to standardize the output of all logs and reports generated by the CLI
+    related to a specific command, and to a specific CI context.
+
+    Note: We cannot hoist this higher in the command hierarchy because only one level of
+    subcommands are available at the time the context is created.
+    """
+
+    git_branch = ctx.obj["git_branch"]
+    git_revision = ctx.obj["git_revision"]
+    pipeline_start_timestamp = ctx.obj["pipeline_start_timestamp"]
+    ci_context = ctx.obj["ci_context"]
+    sanitized_branch = git_branch.replace("/", "_")
+
+    # get the command name for the current context, if a group then prepend the parent command name
+    invoked_subcommand = ctx.invoked_subcommand
+    parent_command_path = ctx.command_path.replace(" ", "/") if ctx.command_path else None
+    cmd = f"{parent_command_path}/{invoked_subcommand}" if parent_command_path else invoked_subcommand
+
+    path_values = [
+        cmd,
+        ci_context,
+        sanitized_branch,
+        pipeline_start_timestamp,
+        git_revision,
+    ]
+
+    # check all values are defined
+    if None in path_values:
+        raise ValueError(f"Missing value required to render the report output prefix: {path_values}")
+
+    # join all values with a slash, and convert all values to string
+    return "/".join(map(str, path_values))
+
+
 # COMMANDS
 
 
 @click.group(help="Commands related to connectors and connector acceptance tests.")
 @click.option("--use-remote-secrets", default=True)  # specific to connectors
+@click.option(
+    "--ci-gcs-credentials",
+    help="The service account to use during CI.",
+    type=click.STRING,
+    required=False,  # Not required for pre-release or local pipelines
```

Comment:
```suggestion
    required=False,  # Not required for pre-release or local pipelines
```
Do we have a logic to validate this is not null when we're on a pre-release?

Reply:
@alafanechere Yup right here: https://github.com/airbytehq/airbyte/blob/bnchrch/dagger/nightly-report-finished-step/tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py#L42